chef/cookbooks/swift/recipes/proxy.rb

```diff
@@ -234,23 +236,26 @@
        action :add_service
      end
 
-     keystone_register "register swift-proxy endpoint" do
-       protocol keystone_settings["protocol"]
-       insecure keystone_settings["insecure"]
-       host keystone_settings["internal_url_host"]
-       auth register_auth_hash
-       port keystone_settings["admin_port"]
-       endpoint_service "swift"
-       endpoint_region keystone_settings["endpoint_region"]
-       endpoint_publicURL "#{swift_protocol}://#{public_host}:"\
-                          "#{node[:swift][:ports][:proxy]}/v1/"\
-                          "#{node[:swift][:reseller_prefix]}$(project_id)s"
-       endpoint_adminURL "#{swift_protocol}://#{admin_host}:"\
-                         "#{node[:swift][:ports][:proxy]}/v1/"
-       endpoint_internalURL "#{swift_protocol}://#{admin_host}:"\
+     # register RadosGW endpoints only if there's no SES based RadosGW present
```

Comment:
```suggestion
     # register swift-proxy endpoints only if no SES based RadosGW is being registered in ses.rb
```
Instead of this comment an alternative would be to call what is done in ses.rb here in an else block.

Reply:
Note that `ses.rb` is in keystone cookbook. Code above will not be called if swift is not deployed.

Reply:
```suggestion
     # register swift-proxy endpoints only if no SES based RadosGW is being registered in keystone/recipes/ses.rb
```
Right, so my alternative won't work. The above is a better fix for the comment.

Reply:
updated. thx!